.tool-versions

```diff
@@ -0,0 +1 @@
+erlang 22.3.4
```

Comment:
FYI. this only affects [asdf](https://github.com/asdf-vm/asdf) users.

Reply:
We used Erlang/OTP 22.3.0 in emqx 4.2.x, should we change it to the latest patch version `22.3.4.13` now?